aten/src/ATen/core/operator_name.h

```diff
@@ -8,6 +8,8 @@ namespace c10 {
 struct OperatorName final {
   std::string name;
   std::string overload_name;
+  OperatorName(const std::string& name, const std::string& overload_name)
```

Comment:
use `OperatorName(std::string` and `std::move` here, otherwise this introduces copies when passing in `const char *` literals